src/account/operations/output_claiming.rs

```diff
@@ -285,7 +285,7 @@ impl AccountHandle {
                     // Recalculate every time, because new intputs can also add more native tokens, which would increase
                     // the storage deposit cost
```

Comment:
```suggestion
                    // Recalculate every time, because new inputs can also add more native tokens, which would increase
                    // the required storage deposit
```